.circleci/config.yml

```diff
@@ -83,6 +84,23 @@ jobs:
       - notify-failure
       - notify-success
 
+  docs_update:
+    executor: default
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/caver-js-ext-kas
+      - check_rc
+      - run:
+          name: "Upload S3 bucket"
+          command: |
+            sudo apt-get update && sudo apt-get install awscli
+            VERSION=$(awk '/"version/ {gsub("\"",""); print $2}' package.json | tr -d ',')
+            if [[ -z $CIRCLE_TAG ]]; then
```

Comment:
Does $CIRCLE_TAG only exist in the production version?

Reply:
CIRCLE_TAG should be like v1.0.0-rc.1, v1.0.0-rc.2. 
so if CIRCLE_TAG not exit, it update to latest directory which means major version release.